src/types.ts

```diff
@@ -72,37 +73,39 @@ export const unallowedChars = /[^A-Za-z0-9]/g;
 
 export const toolDefaults: Transforms = {
   replaceVDomNode: false,
-  variantShapes: true,
+  variantShapes: false,
```

Comment:
Can we flip these back to true for toolDefaults?  These are the settings that are shipped in the elm-optimize-level-2 cli itself, so I just want to be very intentional when they change.

Reply:
Definetly. These have just been flipped for testing, I'll revert that before a potential merge.

Reply:
Is there anything you'd do differently in regards to codegen?